### PR TITLE
Adding Vulnerability Scan

### PR DESCRIPTION
**Pull request by hgarvison**
Adding vulnerability scanning to main pipeline

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -71,6 +71,21 @@ jobs:
...
 
+    - name: Aqua Security Trivy
+      uses: aquasecurity/trivy-action@0.17.0
+      with:
+        scan-type: 'fs'
+        scan-ref: '.'
+        format: 'sarif'
+        output: 'trivy-results.sarif'
+        exit-code: 1
+        severity: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
```
Comment (SethHollandsworth) [inline .github/workflows/ci.yml:82]: Do we want to error out if there's a low severity action found? It seems like that would happen often and may not be resolvable at any given moment

Comment (hgarvison) [reply .github/workflows/ci.yml:82]: As long as it still notifies us of the low severity action, I'm okay with removing it from this list.

Comment (SethHollandsworth) [reply .github/workflows/ci.yml:82]: it looks like there isn't a good way to only exit on High and Critical while still alerting on the lower severities. We could have two separate steps, one to make a report with all vulnerabilities and one that exits on severe vulnerabilities. 

But for now, I think we can trial having all vulnerabilities error out and see how it goes

**Review by ksayid — COMMENTED** [review]
Since vulnerability scanning is being added, should we change the cron job frequency? Right now, I think it's once a week.

**Comment by hgarvison** [comment]
> Since vulnerability scanning is being added, should we change the cron job frequency? Right now, I think it's once a week.

@ksayid are you thinking a daily scan?

**Comment by ksayid** [comment]
@hgarvison Yeah, I was thinking daily, but either would be ok I think. Like the alert level comment, maybe we can trial with daily and see how it goes?

**Review by ksayid — APPROVED** [review]

**Pull request merged**